ui/app/routes/vault/cluster/requests.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
```

Comment:
This should extend `cluster-route-base.js` here so that auth / unseal, etc. checks happen before we land on this route.